src/contrib/testkits/Akka.TestKit.Xunit/Internals/AkkaEqualException.cs

```diff
@@ -9,25 +9,49 @@
 using System.Runtime.Serialization;
 using Xunit.Sdk;
 
-namespace Akka.TestKit.Xunit.Internals
+namespace Akka.TestKit.Xunit2.Internals
```

Comment:
Namespace probably needs to stay the same. We should merge / combine these packages at some point.